ocs_ci/ocs/amq.py

```diff
@@ -569,11 +571,39 @@ def run_amq_benchmark(
         else:
             cmd = "bin/benchmark --drivers /driver_kafka /amq_workload.yaml"
         log.info(f"Run benchmark and running command {cmd} inside the benchmark pod ")
+
+        if run_in_bg:
+            executor = ThreadPoolExecutor(1)
+            result = executor.submit(
+                self.run_amq_workload_in_bg,
+                cmd, benchmark_pod_name, tiller_namespace, timeout
+            )
+            return result
+
         pod_obj = get_pod_obj(name=f"{benchmark_pod_name}-driver", namespace=tiller_namespace)
         result = pod_obj.exec_cmd_on_pod(command=cmd, out_yaml_format=False, timeout=timeout)
 
         return result
 
+    def run_amq_workload_in_bg(
```

Comment:
The function name says we are running the amq workload in bg. where are we using threading inside the function?

Reply:
changed accordingly